SR2019/2019-03-11-competition-information.md

```diff
@@ -50,7 +50,7 @@ With less than 4 weeks left until the final, we hope you're progressing well. Th
 
 ## Kit Loan Extension
 
-In the past, some teams have requested to hold on to the kit for a couple of weeks after the competition. If this is something you would be interested, please let us know on `teams@studentrobotics.org`, with the reason you wish to hold on to the kit. The closing date for this is 31st March.
+In the past, some teams have requested to hold on to the kit for a couple of weeks after the competition. This year, we're allowing a loan extension until 4th May. If this is something you would be interested, please let us know on `teams@studentrobotics.org`, with the reason you wish to hold on to the kit. The closing date for this is 31st March.
```

Comment:
Bear in mind that we'll need time to collate the kits _before_ the Core Team is disbanded. I suggest that that be a week or so?